src/libstd/sys_common/backtrace.rs

```diff
@@ -104,56 +104,52 @@ fn filter_frames(frames: &[Frame],
     // We want to filter out frames with some prefixes
     // from both top and bottom of the call stack.
     static BAD_PREFIXES_TOP: &'static [&'static str] = &[
-        "_ZN3std3sys3imp9backtrace",
+        // std::sys::imp::backtrace
```

Comment:
As I said before, the `a::b::c` forms show up on Windows/MSVC.
Yes, that's probably a demangler problem, but if it's not fixed, then these symbols will not be filtered away.